src/api/v1/db/oracledb/doughs-dao.js

```diff
@@ -265,4 +289,53 @@ const getDoughById = async (id) => {
   }
 };
 
-export { getDoughs, postDough, getDoughById };
+/**
+ * convert object body to an UPDATE query with correct bind params
+ *
+ * @param {object} body a PATCH request body
+ * @returns {object} `bindParams` and `query`
+ */
+const createPatchQueryAndBindParams = (body) => {
+  const inBindparams = getBindParams(body);
+
+  const query = doughsPatchQuery(inBindparams);
+
+  const bindParams = _.assign(
```

Comment:
It's recommended using spread operation over assign according to the Airbnb JS style guide: https://github.com/airbnb/javascript#objects--rest-spread